Jenkinsfile

```diff
@@ -29,7 +29,7 @@ node('docker') {
             dockerCleanup()
             docker.build(image.id, '--pull --no-cache postgres-docker/')
             sh "docker tag ${image.id} registry.devshift.net/${image.id}"
-            docker.withRegistry('https://registry.devshift.net/') {
+            docker.withRegistry('https://push.registry.devshift.net/', 'devshift-registry') {
```

Comment:
We are just pulling images here, so the old registry URL still applies here. Anybody can pull so no credentials are needed. The new `push.registry` URL is just for pushing.

Reply:
In that case we can ignore this PR I believe.

Reply:
Okay, I will update the only have push url modified and keep the pull url as earlier.